SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java

```diff
@@ -2109,6 +2112,31 @@ public void clearInactiveAgent(String host) {
     );
   }
 
+  public Collection<SingularityDeployHistory> getRequestDeployHistory(
```

Comment:
nit: can you make this function name plural so that it's clear that it returns multiple?